serve/internal/api.py

```diff
@@ -35,10 +74,23 @@ async def initialize_logging_on_startup():
 @app.get("/")
 def root():
     logging.info("Root endpoint called")
-    return generate_about_json()
+    return generate_about_json(config)
 
 
 @app.get("/run")
+def run_get(req: Request):
+    return run(req.query_params)
+
+
+@app.post("/run")
+def run_post(rawdata: dict = None):
+    return run(rawdata)
+
+
 def run(rawdata: dict = None):
     logging.info("Run endpoint called")
-    return entrypoint.run(rawdata)
+    if is_app_dev:
+        return serve.run_fake(rawdata)
```

Comment:
I have thoughts on this.
Not thoughts that should block this PR...but let's chat.
Please resolve once read.

Reply:
e.g. if we're going down the OO path (with the `Serve` class) couldn't there be a subclass that delivers fake stuff?